### ISSUE TITLE
User experience challenge - my listings/ my sales

### ISSUE THREAD

**Issue by davecraige** [OriginProtocol/origin#581]
I believe the two separate sections for listings ( /my-listings and the /my-sales) are somewhat confusing.  Aren't users going to be confused by the differences in sales states between these two sections?  How are they going to know to go to /my-sales to see an update of their sales process?  

I made this mistake myself a number of times and was checking /my-listings to see state changes of the items.  I didn't realize that I had to also check the other section to see this.  I believe there may be an opportunity to simplify and improve this.

<img width="155" alt="screen shot 2018-09-26 at 3 09 17 pm" src="https://user-images.githubusercontent.com/2782676/46109547-2e43f800-c19e-11e8-9724-dda5a815b58d.png">

For screenshots see : ✅ https://github.com/OriginProtocol/origin-dapp/issues/580

**Comment by mek32390** [comment]
I believe the confusion stems from the very limited functionality of the "My Listings" page. This page seems to be the predecessor of a "Manage Listings" page, where users could edit their listings. Without this functionality though, the page is confusing because it provides very little utility to the user, yet is the top item of the nav dropdown. However, the functionality to make listings inactive/active is necessary, and this page will likely provide new functionality in the future. When this happens, the page will be given a new life and become less confusing.  

The My Sales page is serving a different purpose but is equally as confusing, because it displays the transaction stage for each listing. This is where the most important information comes from to a seller, and the title, "My Sales", is misleading, because this page displays the stages of the transaction for each listing, and only the final stage of a sold transaction is "Sold". Dispute, Offer, Rejected, and other stages are all displayed on this page, and there are links provided to the page where action can be taken to move the listing to the next stage. This page is definitely the heart of the Seller's experience on the DApp. For this reason, I think this page should really be titled "My Listings" and the functionality to set a listing as active/inactive on the current My Listings page should be transported to this page. There should simply be a tab added for "Inactive" listings and "All listings". The "Pending" text should be "Unsold" and the "Complete" text should be "Sold", to keep things very basic. With these changes, a seller's My Listings page can display listings that are unsold, sold, inactive, and all. 

I would think the pages could be combined in to a single page with relative ease, as there's not much functionality on the "My Listings" page to transport over, but this may not be high priority. These pages could also be approved with a line of copy below the title of the page, or a very basic FAQ page, which I am still a major advocate for!

**Comment by matthewliu** [comment]
I think there were several reasons we kept them separate, and that these reasons might pop back up again.

Not for v1, but:

1. I can create a listing for tickets. I can then sell 100 of these.
2. I can create a listing for a house, rent it out 10 times, then update my listing (though I'm unsure if we will ever do this or just create a new listing). We did talk about preserving state.

I agree the current UX is confusing though.
